.github/workflows/ci.yml

```diff
@@ -34,8 +34,6 @@ jobs:
         run: avocado --version
```

Comment:
I wouldn't mind if this commit gets squashed with the last one, since the last one it is breaking this workflow.